[CI] GenerativeIT test failing

**Build Scans:**
- [elasticsearch-periodic-platform-support #11449 / almalinux-9_platform-support-unix](https://gradle-enterprise.elastic.co/s/hgkrfznhgz3bq)
- [elasticsearch-pull-request #105361 / part-3](https://gradle-enterprise.elastic.co/s/iv4kgparrvmbo)
- [elasticsearch-pull-request #104986 / part-3](https://gradle-enterprise.elastic.co/s/2zj6gmuospofw)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:qa:server:single-node:javaRestTest" --tests "org.elasticsearch.xpack.esql.qa.single_node.GenerativeIT.test" -Dtests.seed=C19CE13F6D1FA65B -Dtests.locale=ff-Adlm-GW -Dtests.timezone=Etc/GMT-5 -Druntime.java=25
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.esql.qa.single_node.GenerativeIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('test'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: query: from date_n*,airports_not_indexed | stats  HrgYUGxU = values(to_upper(abbrev)), `num` = count(*), VReWCvOC = values(scalerank), `city_location` = absent(scalerank + scalerank + scalerank) | inline stats  UXsZNVqS = values(num), yHBrsizQa = sum(num) + avg(num) + median(num), SfXqVAZBr = avg(VReWCvOC) + median(VReWCvOC), city_location = values(HrgYUGxU) by HrgYUGxU
exception: method [POST], host [http://[::1]:39891], URI [/_query?pretty=true&error_trace=true], status line [HTTP/1.1 400 Bad Request]
Warnings: [No limit defined, adding default limit of [1000]]
---
error:
  root_cause:
  - type: "illegal_argument_exception"
    reason: "sub-plan execution results too large [3mb] > 524.2kb"
    stack_trace: "org.elasticsearch.ElasticsearchException$1: sub-plan execution results\
      \ too large [3mb] > 524.2kb\n\tat org.elasticsearch.server@9.3.0-SNAPSHOT/org.elasticsearch.ElasticsearchException.guessRootCauses(ElasticsearchException.java:797)\n\
      \tat
[truncated]
```

**Issue Reasons:**
- [main] 3 failures in test test (1.0% fail rate in 312 executions)
- [main] 2 failures in step part-3 (1.2% fail rate in 167 executions)
- [main] 2 failures in pipeline elasticsearch-pull-request (1.2% fail rate in 172 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -432,9 +432,6 @@ tests:
 - class: org.elasticsearch.test.rest.yaml.CcsCommonYamlTestSuiteIT
   method: test {p0=search/160_exists_query/Test exists query on mapped byte field with no doc values}
   issue: https://github.com/elastic/elasticsearch/issues/138452
-- class: org.elasticsearch.xpack.esql.qa.single_node.GenerativeIT
-  method: test
-  issue: https://github.com/elastic/elasticsearch/issues/138454
 - class: org.elasticsearch.test.rest.yaml.RcsCcsCommonYamlTestSuiteIT
   method: test {p0=search/160_exists_query/Test exists query on geo_point field in empty index}
   issue: https://github.com/elastic/elasticsearch/issues/138461

--- a/x-pack/plugin/esql/qa/server/src/main/java/org/elasticsearch/xpack/esql/qa/rest/generative/GenerativeRestTest.java
+++ b/x-pack/plugin/esql/qa/server/src/main/java/org/elasticsearch/xpack/esql/qa/rest/generative/GenerativeRestTest.java
@@ -59,6 +59,7 @@ public abstract class GenerativeRestTest extends ESRestTestCase implements Query
         "The field names are too complex to process", // field_caps problem
         "must be \\[any type except counter types\\]", // TODO refine the generation of count()
         "INLINE STATS cannot be used after an explicit or implicit LIMIT command",
+        "sub-plan execution results too large",  // INLINE STATS limitations
 
         // Awaiting fixes for query failure
         "Unknown column \\[<all-fields-projected>\\]", // https://github.com/elastic/elasticsearch/issues/121741,

